extras/systemd/foreman.service

```diff
@@ -9,6 +9,7 @@ Type=simple
 User=foreman
 TimeoutSec=300
 WorkingDirectory=/usr/share/foreman
+Environment=MALLOC_ARENA_MAX=2
```

Comment:
@ekohl is asking that you move this to line 15 so that both `Environment=` lines in this file are grouped for readability.

Reply:
Moved.